### ISSUE TITLE
Pace calculations off

### ISSUE THREAD

**Issue by teeks99** [khertan/ForRunners#58]
A couple days ago, I went for a run on a course that I frequent, but hadn't been on with ForRunners yet. During the run, I thought the pace that I was hearing audibly every 5min was a bit off, but assumed it was just me. 

At each 5min audible announcement I had a pace of 9.6-9.8km/hr (I believe this is supposed to be a cumulative pace for the entire run, not an instantaneous pace), including when I reached the 55min point. However, at this time I had already gone 9.9km, so clearly the 9.8km/hr that indicated was off by a few percent.

Here's the GPX file from that run: [20171210_1238.gpx.zip](https://github.com/khertan/ForRunners/files/1557382/20171210_1238.gpx.zip)

**Comment by brvier** [comment]
Yes on older release 1.1.x the pace could be a bit off depending on the gps chip precision. On 1.2.x I rewrote that part of the code to use the same filter everywhere.
